index.html

```diff
@@ -516,6 +570,99 @@ <h3>
               </li>
             </ol>
           </li>
+          <li>If |navigator|.{{Navigator/[[hasGamepadGesture]]}} is `false`,
+          abort these steps.
+          </li>
+          <li>[=list/For each=] |axisIndex:long| of [=the range=] from 0 to the
+          [=list/size=] of |gamepad|.{{Gamepad/axes}} − 1:
+            <ol>
+              <li>If |oldAxisValues|[|axisIndex|] is not equal to
+              |gamepad|.{{Gamepad/[[axes]]}}[|axisIndex|], [=queue a task=] on
+              the [=gamepad task source=] to [=fire an event=] named
+              {{axischange}} at |gamepad| using {{GamepadAxisEvent}} with its
+              {{GamepadAxisEvent/gamepadIndex}} attribute initialized to
+              |gamepad|.{{Gamepad/index}}, its {{GamepadAxisEvent/axisIndex}}
+              attribute initialized to |axisIndex|, its
+              {{GamepadAxisEvent/axisSnapshot}} attribute initialized to
+              |newValue|, and its {{GamepadAxisEvent/gamepadTimestamp}}
+              attribute initialized to |now|.
+              </li>
+            </ol>
+          </li>
+          <li>[=list/For each=] |buttonIndex:long| of [=the range=] from 0 to
+          the [=list/size=] of |gamepad|.{{Gamepad/buttons}} − 1:
+            <ol>
+              <li>Let |button:GamepadButton| be
+              |gamepad|.{{Gamepad/[[buttons]]}}[|buttonIndex|].
+              </li>
+              <li>If |oldButtonValue|[|buttonIndex|] is not equal to
+              |button|.{{GamepadButton/value}}:
+                <ol>
+                  <li>Let |buttonCopy| be a [=new=] {{GamepadButton}} instance
+                  with its {{GamepadButton/value}} attribute initialized to
+                  |button|.{{GamepadButton/value}}, its
+                  {{GamepadButton/pressed}} attribute initialized to
+                  |button|.{{GamepadButton/pressed}}, and its
+                  {{GamepadButton/touched}} attribute initialized to
+                  |button|.{{GamepadButton/touched}}.
+                  </li>
+                  <li>[=Queue a task=] on the [=gamepad task source=] to [=fire
+                  an event=] named {{buttonchange}} at |gamepad| using
```

Comment:
Should `buttonchange` also fire when a button is touched but its value has not changed or do we want to save that for separate `buttontouchdown` and `buttontouchup` events?

Reply:
I think `buttonchange` should only fire for `value` changes and we should add touch events if needed.

The current gamepad market doesn't have many gamepads that detect touch separately from pressure so I think the potential for a gamepad to fire `buttontouchdown` without `buttonchange` is pretty low.

Reply:
Makes sense to me. Do we want to specify that now or leave it as a TODO for later?

Reply:
I am happy either way. I would say let's do the thing that would be easiest from a standards approval perspective. I think that would be to add it here and get it all done in one go.